README.md

```diff
@@ -29,18 +29,31 @@ See https://interledgerjs.github.io/ilp-protocol-psk2
 Uses [`createReceiver`](https://interledgerjs.github.io/ilp-protocol-psk2/modules/_receiver_.html#createreceiver) and [`Receiver.generateAddressAndSecret`](https://interledgerjs.github.io/ilp-protocol-psk2/classes/_receiver_.receiver.html#generateaddressandsecret).
 
 ```js
-const { createReceive } = require('ilp-protocol-psk2')
-const receiver = await createReceiver({
-  plugin: myLedgerPlugin,
-  paymentHandler: async (params) => {
-    // Accept all incoming payments
-    const result = await params.accept()
-    console.log('Got payment for:', result.receivedAmount)
-  }
-})
-
-const { destinationAccount, sharedSecret } = receiver.generateAddressAndSecret()
-// Give these two values to a sender to enable them to send payments to this Receiver
+const crypto = require('crypto')
+const base64url = require('base64url')
+const IlpPluginBtp = require('ilp-plugin-btp')
+const { createReceiver } = require('.')
+
+function tokenToAccount (token) {
+  return base64url(crypto.createHash('sha256').update(token).digest('sha256'))
+}
+
+;(async () => {
+  const receiverToken = base64url(crypto.randomBytes(32))
+  const receiverAccount = tokenToAccount(receiverToken)
```

Comment:
I don't think you need to submit the account (see the example [here](https://github.com/interledgerjs/ilp-plugin-btp#use-as-a-data-channel-for-ilp))

Reply:
Hm, I think that's an incorrect but nonetheless interesting way of using 'user:pass@' in the BTP URI, at least I'm pretty sure that's not what we defined for BTP/1.0. But maybe we can do it that way in BTP/2.0, I'll discuss with @sharafian.

Reply:
I'll update https://github.com/interledger/rfcs/blob/master/0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md#authentication and https://github.com/interledger/interledger/wiki/Interledger-over-BTP#changes-to-setup to accommodate that possibility of using, effectively, bearer tokens instead of username/password-pairs, and then update this example to omit the username. Then maybe also https://github.com/interledgerjs/ilp-plugin-btp/pull/7 will no longer be needed, will see.

Reply:
Yeah, none of the current code requires an account to ever be set, but I think it's a good idea to support it in the URL parsing at least so I approved that PR

Reply:
I included this in https://github.com/interledger/interledger/wiki/Interledger-over-BTP#changes-between-versions-17q4-and-18q1 now.

Reply:
And in https://github.com/interledger/rfcs/pull/372